docker/hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./dev-build-images.sh "totti4ever/"
```

Comment:
Is this on purpose here? Or is it already for the docker-hub integration?

Reply:
Was is testing phase for the docker hub integration, indeed. Is replaced with a generic approach nevertheless now